src/routes/InstancesPage/InstancesPage.js

```diff
@@ -42,6 +42,20 @@ import InstanceDetailsDrawer from './InstanceDetailsDrawer';
 import { getDateTime } from '../../utils/date';
 import Status from '../../components/Status';
 import InstancesToolbarSearchFilter from './InstancesToolbarSearchFilter';
+import useTableSort from '../../hooks/useTableSort';
+
+const sortFields = [
+  'name',
+  'cloud_provider',
+  'region',
+  'owner',
+  'status',
+  'created_at',
+];
+const defaultSortOption = {
+  field: 'name',
+  direction: 'asc',
```

Comment:
`direction: 'asc' | 'desc'` is definitely an improvement over `reversed: true | false`.